build.sbt

```diff
@@ -9,6 +9,7 @@ lazy val freestyle = (crossProject in file("freestyle"))
   .settings(name := "freestyle")
   .jsSettings(sharedJsSettings: _*)
   .settings(libraryDependencies ++= Seq(%("scala-reflect", scalaVersion.value)))
+  .settings(libraryDependencies += "com.47deg" %%% "iota-core" % "0.0.2")
```

Comment:
This should go in the 47deg section of sbt-org-policies instead of hardcoded here so it's automanaged by sbt-dependencies and it makes us aware when versions get outdated